### DISCUSSION TITLE
i cannot use val

### DISCUSSION THREAD

**Discussion by fangfang0304** [mikel-brostrom/yolo_tracking#1260] [Q&A]
hi mikel!!
i want to use val.py,but'**Tracker file not found: D:\yolov8_tracking-master\examples\runs\val\exp1\labels\MOT17-02-FRCNN.txt**',how can i correct it?

**Comment by mikel-brostrom** [comment]
https://github.com/mikel-brostrom/yolo_tracking/issues/1100

**Reply by fangfang0304** [reply]
i want to know if i can input command to solve it,like'--save'?
